### PR TITLE
added notebook for LFQA with PineconeDocumentStore

### PR DESCRIPTION
**Pull request by jamescalam**
for upcoming LFQA article

### CODE REVIEW

File: integrations/haystack/haystack_lfqa.ipynb [hunk 1]
```diff
@@ -0,0 +1,1593 @@
+{
```
Comment (gkogan) [inline integrations/haystack/haystack_lfqa.ipynb:1]: <p>Would be good to link to relevant Pinecone articles in this intro.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gkogan) [inline integrations/haystack/haystack_lfqa.ipynb:1]: <p>Link to relevant docs page within this paragraph. https://www.pinecone.io/docs/integrations/haystack/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (gkogan) [inline integrations/haystack/haystack_lfqa.ipynb:1]: <p>you → You</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Comment by jamescalam** [comment]
@gkogan added those parts

**Pull request merged**